core/templates/dev/head/components/SearchBarDirective.js

```diff
@@ -110,17 +110,28 @@ oppia.directive('searchBar', [function() {
         };
 
         var _searchBarFullyLoaded = false;
+        var url;
```

Comment:
The scope of this is too large -- it's only used in _onSearchQueryChangeExec, so probably better to just declare it there (on line 118)?